diff --git a/android/src/main/java/io/capawesome/capacitorjs/plugins/clipboard/Clipboard.java b/android/src/main/java/io/capawesome/capacitorjs/plugins/clipboard/Clipboard.java
index 08ca3b8a4caa85286158bdd91da477d1ece60ba7..dcdbbb9fc28cd93443aadcd1b65ee2046acf1a0b 100644
--- a/android/src/main/java/io/capawesome/capacitorjs/plugins/clipboard/Clipboard.java
+++ b/android/src/main/java/io/capawesome/capacitorjs/plugins/clipboard/Clipboard.java
@@ -24,7 +24,6 @@ import java.io.InputStream;
 public class Clipboard {
 
     private static final String DEFAULT_LABEL = "";
-    private static final String IMAGE_FILE_NAME = "capawesome_capacitor_clipboard_image.png";
 
     @NonNull
     private final ClipboardPlugin plugin;
@@ -77,7 +76,7 @@ public class Clipboard {
     @NonNull
     private ClipData createImageClipData(@NonNull String label, @NonNull String image) throws Exception {
         byte[] bytes = decodeDataUrl(image);
-        File file = new File(getContext().getCacheDir(), IMAGE_FILE_NAME);
+        File file = File.createTempFile("capawesome-clipboard-", ".png", getContext().getCacheDir());
         try (FileOutputStream outputStream = new FileOutputStream(file)) {
             outputStream.write(bytes);
         }
